app/controllers/orga/seasons_controller.rb

```diff
@@ -45,23 +45,13 @@ def destroy
   end
 
   # # switch_phase: enables developers to easily switch between time dependent settings in views
-  # by opening and closing the corresponding links in the nav bar
+  # by showing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-      case params[:option]
-      when 'Proposals'
-        Season::PhaseSwitcher.fake_proposals_phase
-      when 'Application'
-        Season::PhaseSwitcher.fake_application_phase
-      when 'CodingSummer'
-        Season::PhaseSwitcher.fake_coding_phase
-      when 'RealTime'
-        Season::PhaseSwitcher.back_to_reality
-      end
-    redirect_to orga_seasons_path, notice: "We time travelled into the #{params[:option]} phase"
+    Season::PhaseSwitcher.destined(params[:phase])
```

Comment:
I'm not 100% sure, but doesn't the `phase` end up in the `params` as `string` **always**? So depending on your contract with `PhaseSwitcher` I'd suggest to do the necessary conversion **before** passing it on to the business logic. Could e.g. look like this:

```ruby
def switch_phase
  # ...
  Season::PhaseSwitcher.destined(phase)
end

private

def phase
  params[:phase].to_sym
end
```